services/121-service/src/config.ts

```diff
@@ -1,13 +1,18 @@
-
 export const DEBUG = (['production', 'staging'].indexOf(process.env.NODE_ENV) === -1);
 export const PORT = (process.env.PORT) ? process.env.PORT : 3000;
 export const BASE_PATH = (DEBUG) ? 'api' : '121-service/api';
 export const SCHEME = (DEBUG) ? 'http' : 'https';
 
 const tyknIMS = 'http://11.0.0.3:50001/api/';
 const orgIMS = 'http://11.0.0.4:50002/api/';
+const diberse_url = 'https://b2b.demo.disberse.com/project/​'
```

Comment:
This could be a fine default-value, but make sure you can also set it via a ENV-variable.
So we don't need to change code in a different environment to talk to a different third-party server.